tests/unit/serve/gateway/test_gateway.py

```diff
@@ -227,7 +227,8 @@ def app(self):
             @app.get("/endpoint")
             async def get(text: str):
                 docs = await self.executor['executor1'].post(on='/', inputs=DocumentArray([Document(text=f"{text} {i}") for i in range(20)]), parameters=PARAMETERS, request_size=5)
```

Comment:
make sure to send more documents, otherwise u may not hit all replicas?

Reply:
ah good idea